.github/workflows/ci.yml

```diff
@@ -11,46 +11,56 @@ jobs:
       matrix:
         include:
           - os: ubuntu-22.04
-            rev: nightly/nvim-linux64.tar.gz
+            rev: nightly
             manager: sudo apt-get
             packages: -y ripgrep
           - os: ubuntu-22.04
-            rev: v0.9.0/nvim-linux64.tar.gz
+            rev: v0.9.0
             manager: sudo apt-get
             packages: -y ripgrep
           - os: macos-12
-            rev: nightly/nvim-macos.tar.gz
+            rev: nightly
             manager: brew
             packages: ripgrep
           - os: macos-12
-            rev: v0.9.0/nvim-macos.tar.gz
+            rev: v0.9.0
             manager: brew
             packages: ripgrep
+          - os: windows-2022
+            rev: nightly
+            manager: choco
+            packages: ripgrep
+          - os: windows-2022
+            rev: v0.9.0
+            manager: choco
+            packages: ripgrep
+
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+
       - run: date +%F > todays-date
+
+      - uses: rhysd/action-setup-vim@v1
+        with:
+          neovim: true
+          version: ${{ matrix.rev }}
+
       - name: Restore from todays cache
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
-          path: _neovim
-          key: ${{ runner.os }}-${{ matrix.rev }}-${{ hashFiles('todays-date') }}
+          path: deps
+          key: ${{ runner.os }}-deps-${{ matrix.version }}-${{ hashFiles('todays-date') }}
+          restore-keys: |
+            ${{ runner.os }}-deps-${{ matrix.version }}-
```

Comment:
can we not do this? if cache fails for deps i prefer if we download them again or even better, lets not cache plenary and webicons, we can spare these 4,5 seconds of the clone. it doesnt need to be cached at all. We always wanna run against the last plenary version

Reply:
downloading cache also takes a couple of seconds iirc, so yeah lets just drop this